component/general/FileBrowser.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component, createRef } from 'react';
+import Button from "@material-ui/core/Button/Button";
+import Dialog from "@material-ui/core/Dialog/Dialog";
+import { changeNodeAtPath } from "react-sortable-tree";
+import DialogContent from "@material-ui/core/DialogContent/DialogContent";
+import DialogActions from "@material-ui/core/DialogActions/DialogActions";
+
+import Utils from "../../Utils";
+import Tree from '../../../../js/components/interface/tree/Tree'
+
+
+const styles = {
```

Comment:
I'd avoid inline styles if not forced by some component.

Reply:
In this case, we cannot avoid using that style like that.  I've tried the conventional approach and it did not worked